### PR TITLE
Tv Show Capstone Project

### PR DESCRIPTION
**Pull request by bohaz**
> In this Pull Request we are requesting final approval for the project we have done

- [ ]  Render TV shows from TvMaze API

- [ ]  Create a Pop-up Screen to render specific tv shows from TvMaze Api

- [ ]  create Likes and Comment Counters to track the user interactions with the Tv shows

- [ ] Create a form to submit a comment to the  Involvement API

- [ ] Create a Test for a like and Comment function using Jest and jsdom

- [ ]  Each of these counters was tested using the Jest and Jsdom

- [ ] created a responsive application that adapts to different screen sizes and devices

### Additional Notes

-  In this Project, we have used the GitHub Kanban board to track task progress

-  Followed the best practice of HTML, CSS  and Javascript  and Gitflow

-  All the tasks have been reviewed

## Thank You

### CODE REVIEW

File: src/modules/popUp.js [hunk 1]
```diff
@@ -0,0 +1,268 @@
+/* eslint-disable no-use-before-define */
```
Comment (Deepakdanger) [inline src/modules/popUp.js:1]: - [ ] KIndly remove this & solve the linter manually, if the error present.

File: src/modules/fetchLikes.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (Deepakdanger) [inline src/modules/fetchLikes.js:1]: - [ ] KIndly remove this & solve the linter manually, if the error present.

File: src/modules/api.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (Deepakdanger) [inline src/modules/api.js:1]: - [ ] KIndly remove this & solve the linter manually, if the error present.

File: src/modules/postLikes.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (Deepakdanger) [inline src/modules/postLikes.js:1]: - [ ] KIndly remove this & solve the linter manually, if the error present.

File: src/modules/helpers.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/prefer-default-export */
```
Comment (Deepakdanger) [inline src/modules/helpers.js:3]: - [ ] KIndly remove this & solve the linter manually, if the error present.

File: README.md [hunk 6]
```diff
@@ -61,231 +30,178 @@ For more information on the importance of a professional README for your reposit
...
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Capstone-Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Capstone-Project]** This is a collaborative project where we have applied all the skills obtained throughout the javascript module.
```
Comment (Deepakdanger) [inline README.md:37]: - [ ] Give a proper name to the project "Capstone-Project" is not sound good,& give a proper description of the project not about collaboration.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @bohaz & @henask12 , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- Website looks professional :+1: 
- all test cases are passing
- Nice presentation, keep it up
- Good use of ES6

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would be nice if the latest comment should display on the first page.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/popUp.js (hunk 1 above)
Comment (rloterh) [reply src/modules/popUp.js]: well resolved! ✔️

File: src/modules/fetchLikes.js (hunk 2 above)
Comment (rloterh) [reply src/modules/fetchLikes.js]: well resolved! ✔️

File: src/modules/api.js (hunk 3 above)
Comment (rloterh) [reply src/modules/api.js]: well resolved! ✔️

File: src/modules/helpers.js (hunk 5 above)
Comment (rloterh) [reply src/modules/helpers.js]: well resolved! ✔️

**Comment by rloterh** [comment]
Hi @henask12 

Also, find a suitable title for your project, as mentioned by the previous reviewer

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ricardo & Hena,  

In my opinion, your project is ready for presentation! Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲  Nice job improving upon code readability and ReadMe!

Cheers..Keep rocking! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**